### ISSUE TITLE
Cannot run Ren'Py project on iOS simulator

### ISSUE THREAD

**Issue by robertpeacock22** [renpy/renpy#1366]
## Environment
Ren'Py version: 6.99.14.3135
Xcode version: 8.2.1 (8C1002)
Simulator version: 10.0 (SimulatorApp-726.7)
Simulated device: iPhone 7 Plus
Mac OS version: 10.11.6 (El Capitan)

## Steps to reproduce
1. Open built-in project "The Question" (or theoretically any project that is known to otherwise work)
2. Select `iOS` > `Create Xcode Project`
3. Open resulting project in Xcode and run in simulator

## Result
You will see a "created with Ren'Py" loading screen, followed by a crash and a black screen. The following output is given in Xcode:
```
running /Users/robert/Library/Developer/CoreSimulator/Devices/761BE1B0-DCC2-457D-AD5B-F90797017DB3/data/Containers/Bundle/Application/46B310E2-2DAC-40E8-88AF-FEFAFB2D165F/the_question.app/base/main.py
Got dlopen error on Foundation: dlopen(/System/Library/Frameworks/Foundation.framework/Versions/Current/Foundation, 1): no suitable image found.  Did find:
	/System/Library/Frameworks/Foundation.framework/Versions/Current/Foundation: mach-o, but not built for iOS simulator
Got fallback dlopen error on Foundation: dlopen(/Groups/System/Library/Frameworks/Foundation.framework/Versions/Current/Foundation, 1): image not found
2018-01-26 18:46:41.467 the_question[1937:27141] Traceback (most recent call last):
2018-01-26 18:46:41.467 the_question[1937:27141]   File "/Users/robert/Library/Developer/CoreSimulator/Devices/761BE1B0-DCC2-457D-AD5B-F90797017DB3/data/Containers/Bundle/Application/46B310E2-2DAC-40E8-88AF-FEFAFB2D165F/the_question.app/base/main.py", line 198, in <module>
2018-01-26 18:46:41.468 the_question[1937:27141]     main()
2018-01-26 18:46:41.468 the_question[1937:27141]   File "/Users/robert/Library/Developer/CoreSimulator/Devices/761BE1B0-DCC2-457D-AD5B-F90797017DB3/data/Containers/Bundle/Application/46B310E2-2DAC-40E8-88AF-FEFAFB2D165F/the_question.app/base/main.py", line 195, in main
2018-01-26 18:46:41.468 the_question[1937:27141]     renpy.bootstrap.bootstrap(renpy_base)
2018-01-26 18:46:41.469 the_question[1937:27141]   File "/Users/robert/Library/Developer/CoreSimulator/Devices/761BE1B0-DCC2-457D-AD5B-F90797017DB3/data/Containers/Bundle/Application/46B310E2-2DAC-40E8-88AF-FEFAFB2D165F/the_question.app/base/renpy/bootstrap.py", line 281, in bootstrap
2018-01-26 18:46:41.469 the_question[1937:27141]     renpy.import_all()
2018-01-26 18:46:41.469 the_question[1937:27141]   File "/Users/robert/Library/Developer/CoreSimulator/Devices/761BE1B0-DCC2-457D-AD5B-F90797017DB3/data/Containers/Bundle/Application/46B310E2-2DAC-40E8-88AF-FEFAFB2D165F/the_question.app/base/renpy/__init__.py", line 357, in import_all
2018-01-26 18:46:41.470 the_question[1937:27141]     import renpy.pyanalysis
2018-01-26 18:46:41.470 the_question[1937:27141]   File "/Users/robert/Library/Developer/CoreSimulator/Devices/761BE1B0-DCC2-457D-AD5B-F90797017DB3/data/Containers/Bundle/Application/46B310E2-2DAC-40E8-88AF-FEFAFB2D165F/the_question.app/base/renpy/pyanalysis.py", line 28, in <module>
2018-01-26 18:46:41.470 the_question[1937:27141]     from renpy.python import py_compile
2018-01-26 18:46:41.471 the_question[1937:27141]   File "/Users/robert/Library/Developer/CoreSimulator/Devices/761BE1B0-DCC2-457D-AD5B-F90797017DB3/data/Containers/Bundle/Application/46B310E2-2DAC-40E8-88AF-FEFAFB2D165F/the_question.app/base/renpy/python.py", line 86, in <module>
2018-01-26 18:46:41.471 the_question[1937:27141]     from renpy.pydict import DictItems, find_changes
2018-01-26 18:46:41.471 the_question[1937:27141] ImportError: dynamic module does not define init function (initpydict)

```

**Comment by robertpeacock22** [comment]
I tried upgrading Mac OS to High Sierra and upgrading Xcode to 9.2, but I'm still having the same issue.

**Comment by robertpeacock22** [comment]
I'm having the same issue on a different Mac, using Sierra and Xcode 9.2 (still Ren'Py 6.99.14.3135 though).

**Comment by robertpeacock22** [comment]
This looks like it relates to https://github.com/kivy/pyobjus/issues/36, for which no solution appears to have been found.

**Comment by robertpeacock22** [comment]
I got my hands on an iPad (iOS 10.3.3, iPad Air late 2013) and the app crashes in exactly the same way at startup when I run it on the device.

**Comment by robertpeacock22** [comment]
Fixed in Feb 4th release of 6.99.14.1. Thanks @renpytom!
